db/columns/operations/infer_types.py

```diff
@@ -65,7 +65,7 @@ def infer_column_type(schema, table_name, column_name, engine, depth=0, type_inf
     for type_str in type_inference_dag.get(column_type_str, []):
         try:
             with engine.begin() as conn:
-                alter_column_type(table, column_name, engine, conn, type_str)
+                alter_column_type(get_oid_from_table(table_name, schema, engine), column_name, engine, conn, type_str)
```

Comment:
Could you unnest the `get_oid_from_table` call into an intermediary variable named something like `table_oid`?